pyccel/ast/fortran.py

```diff
@@ -1,4 +1,9 @@
 # coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder.

I think this file needs deleting. Nothing in it is used and everything has an equivalent elsewhere. @saidctb do you agree?

Reply:
yes I agree we can delete it